scripts/checkCoreLicenses.sh

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
```

Comment:
The solution works fine so the procedure is good, but I think it's worth to re-factor